has_files.sh

```diff
@@ -39,8 +39,9 @@ process_zip () {
 process_7z () {
   mkdir -p "${user_folder}/hashes/${rom_path}/tmp"
   echo "unzipping ${file}"
-  7z x "${user_folder}${rom_path}/${file}" -o"${user_folder}/hashes/${rom_path}/tmp"
+  7z e "${user_folder}${rom_path}/${file}" -o"${user_folder}/hashes/${rom_path}/tmp"
   rm "${user_folder}/hashes/${rom_path}/tmp/"*.{txt,nfo,xml,readme,README} &> /dev/null || :
+  find "${user_folder}/hashes/${rom_path}/tmp/" -empty -type d -delete
```

Comment:
Another option would be:

```suggestion
rmdir "${user_folder}/hashes/${rom_path}/tmp/*" --ignore-fail-on-non-empty
```

That might be cleaner.